src/snakefile

```diff
@@ -76,251 +77,322 @@ osemosysFiles = [
 ## TARGETS
 #####################################
 
-configfile: '../scripts/config.yaml'
+configfile: Path('..', 'scripts', 'config.yaml')
 fileName = 'CanadaUSA'
-inputDir = '../dataSources/'
-scriptsDir = '../scripts/'
-outputDir = '../src/'
+inputDir = Path('..','dataSources')
+scriptsDir = Path('..','scripts')
+outputDir = Path('..', 'src')
 canadaProvinces = list(itertools.chain.from_iterable(
 	config['regions_dict']['CAN'].values()))
 
 rule all:
 	input: 
-		outputDir + fileName + '.txt'
+		f'{fileName}.txt'
 
-rule dataFile:
+rule data_file:
 	input:
-		outputDir + fileName + '.txt'
+		f'{fileName}.txt'
 
-rule lpFile:
-	input: 
-		outputDir + fileName + '.lp'
+#rule lp_file:

```

Comment:
Intentionally commented out lpFile? Maybe it should be deleted instead?

Reply:
Good point. Its going to be reintroduced with issue #80, so I left it in for the time being tho?

Reply:
No I think it's fair enough to leave it commented right now, because of that issue and because it doesn't work when uncommented.